user_guide_src/source/models/entities/021.php

```diff
@@ -9,11 +11,11 @@ public static function get($value, array $params = [])
         var_dump($params);
         // array(3) {
         //   [0]=>
-        //   string(13) "App\SomeClass"
+        // string(13) "App\SomeClass"
         //   [1]=>
-        //   string(6) "param2"
+        // string(6) "param2"
         //   [2]=>
-        //   string(6) "param3"
+        // string(6) "param3"
```

Comment:
Removing spaces is not good.
Does cs-fixer these?

Reply:
Yes thats the fixer. I had it turned on in my IDE during this PR, so some of the files are already compliant.

However, I manually fixed the spaces now. We will need to deal with this in the PR that applies the fixer then.

Reply:
I think some fixer rules will need to be disabled for the example, such as removing empty lines after single-line comments.

Reply:
Yes.

@paulbalandan How do we disable the rule `Nexus/space_after_comment_start`?

Reply:
I think we will create a new cs-config file for the docs and edit the custom rules for that:
https://github.com/codeigniter4/CodeIgniter4/blob/8211a86813a6058bdf8b09853d9a714ac3fec015/.no-header.php-cs-fixer.dist.php#L39-L42

We will see which other rules might have to be disabled then.

Reply:
To disable a built-in rule, just set the value to `false`. To disable a custom rule, just don't include it in the list of custom rules.

I think a new config file specifically for the user guide should be the right way to go, because cs-fixer does not cater per-file disabling of rules.

Reply:
I sent PR #5765